test/specs/appLog.spec.js

```diff
@@ -20,14 +30,15 @@ const getLogEventOptions = (message, facilityCode) => {
     dim1: middlewareSourceCode,
     dim2: noneViewName,
     dim3: deviceType,
-    dim4: getCurrentTimeOfDayDimValue()
+    dim4: nightOrDay(),
   };
 };
 
 describe('Logging API Tests', () => {
   const client = factory({
     appKey: '56ea6a370db1bf032c9df5cb',
     deviceId: 'gregTestingSDK',
+    // log(...args) { console.log(...args); },
```

Comment:
I left some of those in tests only, makes it easy to see what happens if a test fails. We don't want any of those outside of tests though